lang/src/lib.rs

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU General Public License
 // along with ink!.  If not, see <http://www.gnu.org/licenses/>.
 
-#![recursion_limit = "256"]
+#![recursion_limit = "512"]
```

Comment:
Oh my ... 0.o why was that?

Reply:
I really hope that dtolnay releases a proc-macro version of `quote` soon.

Reply:
Indeed! Can't remember what it was...in any case I've reverted it and it seems to work okay now

Reply:
Reverted